frontend/src/metabase/ui/components/inputs/Select/SelectDropdown/SelectDropdown.tsx

```diff
@@ -17,5 +18,12 @@ export const SelectDropdown = forwardRef(function SelectDropdown(
     return () => removeCloseHandler();
   }, [setupCloseHandler, removeCloseHandler]);
 
-  return <div data-testid="select-dropdown" ref={ref} {...props} />;
+  return (
+    <Box
+      data-testid="select-dropdown"
+      ref={ref}
+      display="contents"
```

Comment:
checked [caniuse](https://caniuse.com/?search=display%3A%20contents) for this and it should be fine.